README.md

```diff
@@ -1 +1,145 @@
-# Capstone_project
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone Project] <a name="about-project"></a>
+
+
+**Capstone Project** is a personal projects which i have made during microverse studies at the end of the first module
+
+## 🛠 Built With <a name="built-with"></a>
+
+    .Webhint
+    .Styleint
+    .html
+    .css
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **adding some emojis to the html file.**
+- **adding some more interesting background to webpage**
+- **adding more semantic tags to make it rwd**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [REQUIRED] Your should make use of `GitHub Pages` to deploy the web pages, the project should deployed properly, all pages, links, and images work as expected and a link to the deployed project is present in the `README` 🙏🏼